integration-tests/reactive-messaging-hibernate-orm/src/main/java/io/quarkus/it/kafka/fruit/FruitDto.java

```diff
@@ -1,5 +1,8 @@
 package io.quarkus.it.kafka.fruit;
 
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
```

Comment:
Is this requirement new or was it an oversight?

Reply:
It isn't a new requirement. It was missed before because this IT isn't included in the native tests. But I don't think we need to add it.